app/src/main/java/org/wikipedia/readinglist/ReadingListItemView.kt

```diff
@@ -199,6 +199,7 @@ class ReadingListItemView : ConstraintLayout {
 
     private inner class OverflowMenuClickListener constructor(private val list: ReadingList?) : PopupMenu.OnMenuItemClickListener {
         override fun onMenuItemClick(item: MenuItem): Boolean {
+            BreadCrumbLogEvent.logMenuItemSelection(context, item.title.toString())
```

Comment:
The `item.title` will give you the translated text of the menu item, which we don't want. We want the resource name of the menu item, which you can obtain by using `resources.getResourceEntryName(item.itemId)`.